Makefile

```diff
@@ -66,6 +66,18 @@ test:
 test-pyquil:
 	pytest -v --cov=mitiq --cov-report=term --cov-report=xml mitiq/mitiq_pyquil
 
+.PHONY: test-qiskit
+test-qiskit:
+	pytest -v --cov=mitiq --cov-report=term --cov-report=xml mitiq/mitiq_qiskit
+
+.PHONY: test-cirq
+test-cirq:
+	pytest -v --cov=mitiq --cov-report=term --cov-report=xml mitiq/mitiq_cirq
+
+.PHONY: test-braket
+test-braket:
+	pytest -v --cov=mitiq --cov-report=term --cov-report=xml mitiq/mitiq_braket
+
```

Comment:
We decided elsewhere to only keep `make test-pyquil` because for pyquil you have to have a separate program running. So, you can remove all of these except `make test-pyquil`, `make test`, and `make test-all`.